test/Microsoft.AspNet.Mvc.TagHelpers.Test/FormTagHelperTest.cs

```diff
@@ -257,6 +257,49 @@ public class FormTagHelperTest
             Assert.Empty(output.PostContent.GetContent());
         }
 
+        [Fact]
+        public async Task ProcessAsync_CallsIntoGenerateRouteFormWithExpectedParameters()
+        {
+            // Arrange
+            var viewContext = CreateViewContext();
+            var context = new TagHelperContext(
+                allAttributes: new Dictionary<string, object>(),
+                items: new Dictionary<object, object>(),
+                uniqueId: "test",
+                getChildContentAsync: () =>
+                {
+                    var tagHelperContent = new DefaultTagHelperContent();
+                    tagHelperContent.SetContent("Something");
+                    return Task.FromResult<TagHelperContent>(tagHelperContent);
+                });
+            var output = new TagHelperOutput(
+                "form",
+                attributes: new Dictionary<string, object>());
```

Comment:
Add some `asp-route-*` attributes to verify they're passed through to `GenerateRouteForm`.